app/data_access/control_data.py

```diff
@@ -212,6 +220,19 @@ class Meta:
     reported_infractions_last_update_time = graphene.Field(
         TimeStamp, required=False
     )
+    can_take_pictures = graphene.Field(
+        graphene.Boolean,
+        required=False,
+        description="Indique s'il est possible de prendre des photos pour un LIC papier.",
+    )
+    pictures = graphene.List(
+        ControlPictureOutput,
+        description="Liste des photos rattachées au contrôle LIC papier.",
```

Comment:
suggestion(non-blocking): we could tell here that it is presigned urls?

Reply:
ok, I will add a description to the url field